bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/GmosRecipe.java

```diff
@@ -466,4 +473,12 @@ private static Color ccdDarkColor(final int ccdIndex) {
             default: throw new Error();
         }
     }
+
+    // In the multi-CCD case we have to force the first and last y values to 0 to cancel out signal and s2n spikes
+    // caused by interpolation, resampling or some other effect of how data is calculated around CCD gaps.
+    // TODO: DetectorTransmissionVisitor needs a serious overhaul so that this behavior becomes more predictable.
+    private static void fixGapBorders(double[][] data) {
+        data[1][0]                  = 0.0;
```

Comment:
I guess this is the `hack`. It doesn't look that bad, in many cases you need to fix border conditions like this. At least I don't have a big problem with this